core/test/equations.jl

```diff
@@ -145,3 +145,25 @@ end
 
     @test all(isapprox.(LHS, RHS; rtol = 0.005)) # Fails with '≈'
 end
+
+# Simple solutions: 
+# storage1 = storage1(t0) + (t-t0)*(frac*q_boundary - q_pump)
+# storage2 = storage2(t0) + (t-t0)*q_pump
+# Note: uses Euler algorithm
```

Comment:
I feel we should discuss if we find the errors of the default solver acceptable.
If yes, that's also the solver we should use for this test. Agreed @visr?